src/wasm.h

```diff
@@ -1427,15 +1427,15 @@ class Function {
   }
 };
 
+enum class Kind {
```

Comment:
Yeah, design repo uses `external_kind` for the type, but uses `kind` for the name which is more frequent - and while we are discussing the type here, that they can use `kind` for the name suggests no conflict.

Good point about `GlobalKind`, since there is a `Global`... so I'm not sure what to do here.